venues/DBLP.org/python/dblp_wrapper.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+
+"""
+
+This is the initialization script for dblp.org
+added dblp directory and tutorial section
+
+It should only be run ONCE to kick off the conference. It can only be run by the Super User.
+
+"""
```

Comment:
could you change this docstring (or remove it entirely) and rename the file to something more descriptive?

maybe "dblp_upload.py", or something like that?